src/wp-admin/includes/media.php

```diff
@@ -3237,16 +3237,17 @@ function edit_form_image_editor( $post ) {
 		<?php
 
 		printf(
-			/* translators: 1: Link to tutorial, 2: Additional link attributes, 3: Accessibility text. */
-			__( '<a href="%1$s" %2$s>Learn how to describe the purpose of the image%3$s</a>. Leave empty if the image is purely decorative.' ),
+			/* translators: 1: Link to tutorial, 2: Additional link attributes, 3: Accessibility text, 4: External link icon. */
+			__( 'Alt text describes the image\'s purpose. Leave it blank if the image is purely decorative. <a href="%1$s" %2$s>Learn more about alt text%3$s</a>.%4$s' ),
```

Comment:
The right single quotation mark is more commonly used as an apostrophe for possessives in core's translated strings:

```suggestion
			__( 'Alt text describes the image&#8217;s purpose. Leave it blank if the image is purely decorative. <a href="%1$s" %2$s>Learn more about alt text%3$s</a>.%4$s' ),
```